src/guides/v2.3/javascript-dev-guide/widgets/widget_modal.md

```diff
@@ -385,6 +385,45 @@ The following example shows how to initialize the modal widget and pass options
 </div>
 ```
 
+The following example shows a PHTML file using the script:
+
+```html
+<button id="button">Click me</button>
```

Comment:
Would suggest encapsulating string literals in the PHTML with `$block->escapeHtml(__())` as this is best practice

Reply:
Hi @BarnyShergold 
Thanks for your notice.
Your suggestion have been implemented!
Could you please verify this PR?
Thank you!